src/test/scala/cornichonTests/CheckoutTest.scala

```diff
@@ -0,0 +1,128 @@
+package cornichonTests
+import com.github.agourlay.cornichon.core.FeatureDef
+
+class CheckoutTest extends FeatureWithToken {
+
+  def feature: FeatureDef = Feature("Checkout process") {
+    Scenario("Adding item to cart | Making order from cart | Deleting order, cart, tax") {
+      WithToken {
+        Then assert createCart
+        Then assert createTaxCategory
+        Then assert addCustomLineItem
+        Then assert createOrderFromCart
+        Then assert getCartById
+        Then assert deleteOrderFromCart
+        Then assert deleteCart
+        Then assert deleteTaxCategory
+        And I show_last_body_json
```

Comment:
In general you can remove the `show_last_body_json`. It's more useful when creating the tests.
Once they are ready, it usually adds noise to the output.

Reply:
got it, will remove.

Reply:
Do you mean changing this test to check for an existing product or creating a new test?

Reply:
as you want. But we can go with another test so that we can merge this one.

Reply:
Ok, will do that in a new test then